core-bundle/tests/Twig/Loader/ContaoFilesystemLoaderTest.php

```diff
@@ -664,7 +664,7 @@ private function mockFilemtime(array $pathToMtime): void
                     namespace %s;
 
                     function filemtime(string $filename) {
-                        if (null !== ($mtime = unserialize('%s')[\Webmozart\PathUtil\Path::canonicalize($filename)] ?? null)) {
+                        if (null !== ($mtime = unserialize('%s')[\Symfony\Component\Filesystem\Path::canonicalize($filename)] ?? null)) {
```

Comment:
Why are we using a FQCN here?

Reply:
It's part of a string. We could also write an import, but this looks simpler to me.

Reply:
> It's part of a string.

Really? It seems like an array key to me. 🤔

Reply:
@leofeyer the whole code block is part of a string, see line 663: `<<<'EOPHP'`